packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -585,11 +585,7 @@ export const mutation = mutationType({
         projectId: nonNull(stringArg()),
       },
       resolve: async (_, args, ctx) => {
-        try {
-          await ctx.actions.project.setProjectIdInConfigFile(args.projectId)
-        } catch {
-          // ignore error as not useful for end user to see
-        }
+        await ctx.actions.project.setProjectIdInConfigFile(args.projectId)
```

Comment:
hmmm is a way to catch this and provide an useful error message to the end-user so the error isn't hidden while giving the user information on where to find help or how to solve it?

Reply:
We end up showing two different errors different places, one of which is the launchpad, which does not seem ideal. @tgriesser is this expected? I only found this error when I went looking for the little blue toast to restart Cypress.

![Screen Shot 2022-04-05 at 4 19 01 PM](https://user-images.githubusercontent.com/8340719/161842066-bb1b0e68-ea5c-4ecf-b63e-a5f86d26980f.png)

![Screen Shot 2022-04-05 at 4 17 06 PM](https://user-images.githubusercontent.com/8340719/161842075-2d06b08c-619a-4354-9ec9-adde26a46a1c.png)

Reply:
Yeah, we probably need to sync on this and figure out how to make this actionable as part of the response of the mutation, rather than having this be unhandled and displayed like this.

Any error in a mutation should be caught/handled, otherwise it shows up as the "An unexpected internal error occurred" as above, which we don't really want. Maybe we want to log the error out to the console or something? 

I think the way the error case handled is a bit hard to follow in this flow and could be improved, it's almost like we want the return type of the mutation to be a union of `Query | ErrorWrapper`